### PR TITLE
DOCS-10906 - Add Compass Installer to MongoDB packaging

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
Code review: https://mongodbcr.appspot.com/173770002/

Staging: 

https://docs-mongodbcom-staging.corp.mongodb.com/jeffreyallen/DOCS-10906/installation.html

https://docs-mongodbcom-staging.corp.mongodb.com/jeffreyallen/DOCS-10906/tutorial/install-mongodb-on-windows.html

---
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/3130)

### CODE REVIEW

File: source/installation.txt [hunk 1]
```diff
@@ -23,6 +23,15 @@ MongoDB.
...
 For upgrade instructions, see :ref:`3.6-upgrade` instead.
 
+.. note::
+
+   .. versionadded:: 3.6
+
+      Each MongoDB installer is packaged with a platform-specific
+      :ref:`MongoDB Compass <compass-index>` installation script. This
+      script installs MongoDB Compass as part of the MongoDB Server
+      installation process.
```
Comment (kay-kim) [inline source/installation.txt]: we'll actually need to include a install_compass for the installation script under
https://docs.mongodb.com/master/reference/program/

Then, basically, this new page just state that this is the installer for Compass Community edition and how to run.

Then link to: https://docs.mongodb.com/compass/current/#compass-and-compass-community-editions

**Comment by jeff-allen-mongo** [comment]
Hey @kay-kim, I just updated the /reference/program copy

File: source/reference/program.txt [hunk 2]
```diff
@@ -112,6 +112,15 @@ GridFS
...
 
+MongoDB Compass
+---------------
+
+.. versionadded:: 3.6
+
+   MongoDB is packaged with an ``install_compass`` script, which
+   is a platform-specific installer for
+   :ref:`MongoDB Compass Community Edition <compass-index>`.
+
```
Comment (kay-kim) [inline source/reference/program.txt]: We'll need:

.. toctree::
   :maxdepth: 1

   /reference/program/install_compass

(source/reference/program/install_compass.txt would have:
.. program:: install_compass

   .. versionadded:: 3.6

   Platform-specific installer for :ref:`MongoDB Compass Community Edition <compass-index>`.

   To run, on Linux/iOS:

   .. code-block:: sh

       python install_compass

   To run, on Windows:

(we might have to check on our Windows box tomorrow).

      
We also should blurb in  3.6 rel notes.

And definitely, stop working :D  It's not that urgent.

**Comment by jeff-allen-mongo** [comment]
Hey @kay-kim, I added a page for the ``install_compass`` script including the instructions for Linux / MacOS and Windows. Also updated the rel notes.

**Pull request merged**